app/controllers/socure_webhook_controller.rb

```diff
@@ -52,18 +53,46 @@ def verify_queue(authorization_header:)
   end
 
   def log_webhook_receipt
-    event = socure_params[:event]
+    @event = socure_params[:event]
 
     analytics.idv_doc_auth_socure_webhook_received(
-      created_at: event[:created],
-      customer_user_id: event[:customerUserId],
-      event_type: event[:eventType],
-      reference_id: event[:referenceId],
-      user_id: event[:customerUserId],
+      created_at: @event[:created],
+      customer_user_id: @event[:customerUserId],
+      event_type: @event[:eventType],
+      reference_id: @event[:referenceId],
+      user_id: @event[:customerUserId],
+    )
+  end
+
+  def process_webhook_event
+    case @event[:eventType]
+    when 'DOCUMENTS_UPLOADED'
+      increment_rate_limiter
+    end
+  end
+
+  def increment_rate_limiter
+    @document_capture_session = DocumentCaptureSession.find_by(
+      user_id: @event[:customerUserId],
+      socure_docv_transaction_token: @event[:docvTransactionToken],
+    )
+    if !@document_capture_session.nil?
+      rate_limiter.increment!
+    end
+    # TODO: check to throw error here
```

Comment:
Will this TODO be tackled in this pull request, or is there a separate ticket for it?

Reply:
Good question, I put TODOs, to see if my team would like to ticket these issues (because I don't see tickets for them yet). Is not part of this ticket/PR. Should these be removed?